code/modules/surgery/organs/eyes.dm

```diff
@@ -186,37 +186,70 @@
 
 	damaged = TRUE
 
+#define NIGHTVISION_LIGHT_OFF 0
+#define NIGHTVISION_LIGHT_LOW 1
+#define NIGHTVISION_LIGHT_MID 2
+#define NIGHTVISION_LIGHT_HIG 3
+
 /obj/item/organ/internal/eyes/night_vision
-	see_in_dark = NIGHTVISION_FOV_RANGE
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	// These lists are used as the color cutoff for the eye
+	// They need to be filled out for subtypes
+	var/list/low_light_cutoff
+	var/list/medium_light_cutoff
+	var/list/high_light_cutoff
+	var/light_level = NIGHTVISION_LIGHT_OFF
 	actions_types = list(/datum/action/item_action/organ_action/use)
```

Comment:
cleanliness nit, put `action_types` before the new var declarations.

Reply:
sure